### ISSUE TITLE
[DOCUMENTATION]: Create Pull Request Template

### ISSUE THREAD

**Issue by amirhessam88** [slickml/slick-ml#93]
### Describe the documentation improvements you are interested in ...

Currently, we dont have any template for PRs. Once should be go0od here. Something simple similar to `poetry` https://github.com/python-poetry/poetry/blob/master/.github/PULL_REQUEST_TEMPLATE.md

**Comment by amirhessam88** [comment]
@chadac What are you thoughts on a pattern for PR's title? I am thinking of `Add: blah bah ....`, `Fix: blah blah`; so we can keep it simple, and these can be easily reflected in the release notes. What do you think ?

**Comment by chadac** [comment]
@amirhessam88 Yeah that's a good question. I'm all for prefixes like that, it's a convention that is common on some other repos although not enforced consistently. Stuff liks `Docs: blah` or `Typing: blah` or `Lint: blah` I've also seen.

Let me see if GitHub has any way to make title templates like that easy.
